### PR TITLE
Small fixes & suggestions

### PR DESCRIPTION
**Pull request by sloede**
Thanks for the great idea and the effort you put into this! This manifesto encompasses many thoughts I have reiterated in this form or another, but in a succinct and _shareable_ manner - great work!

I took the liberty to suggest a few small improvements / fixes. Please let me know if anything is unclear.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Keeping it simple and with academia in mind, the most fundamentally important as
 
 Now we may ask ourselves: _"What is the ideal programming language?"_
 
-It is the one that is the best in all of the aforementioned fundamental aspects. This language does not exist as pros-and-cons are an integral part of the real world. Therefore, the best _real_ language is the one that overall accumulates the highest quality when explored across these four fundamental aspects.
+It is the one that is the best in all of the aforementioned fundamental aspects. This language does not exist as pros-and-cons are an integral part of the real world. Therefore, the best _real_ language is the one that overall accumulates the highest quality when explored across these five fundamental aspects.
```
Comment (Datseris) [inline README.md]: Could you just remove the count? instead of saying five, just not add a word!

**Comment by Datseris** [comment]
Ah, actually, I have already done that. If you updated to current `main` btranch you will see i have corrected the four -> five mistake.

(The rest of your changes I am happy to merge in!)

File: README.md (hunk 1 above)
Comment (sloede) [inline README.md]:
```suggestion
It is the one that is the best in all of the aforementioned fundamental aspects. This language does not exist as pros-and-cons are an integral part of the real world. Therefore, the best _real_ language is the one that overall accumulates the highest quality when explored across these fundamental aspects.
```

Comment (sloede) [reply README.md]: Sorry, fixed in [3650686](https://github.com/Datseris/whyjulia-manifesto/pull/6/commits/36506864ea91ffd53a194f233077443c695966fa).

**Pull request merged**